divide_and_conquer/max_difference_pair.py

```diff
@@ -1,7 +1,7 @@
-from typing import List
+from typing import List, Tuple
```

Comment:
@dhruvmanila Do we need these imports for basic types in Python 3.9 and the current mypy?

Reply:
No, I updated the issue with the message: https://github.com/TheAlgorithms/Python/issues/4052#issuecomment-783919171

Reply:
I see, I made the corresponding changes. One difference is that the basic types are spelled lower case with this import